frontend/app-development/features/dataModelling/SchemaEditorWithToolbar/SelectedSchemaEditor.test.tsx

```diff
@@ -83,30 +88,29 @@ describe('SelectedSchemaEditor', () => {
     expect(saveDatamodel).toHaveBeenCalledWith(org, app, modelPath, dataMock);
   });
 
-  it('Autosaves when changing between models that are not present in the cache', async () => {
+  test('Autosaves when changing between models that are not present in the cache', async () => {
```

Comment:
`it` og `test` er samme funksjon, som har to forskjellige navn kun for å kunne forme sammenhengende setninger sammen med funksjonsnavnet. Her går testnavnet fint sammen med `it`.
```suggestion
  it('Autosaves when changing between models that are not present in the cache', async () => {
```